util/reggen/data.py

```diff
@@ -228,12 +230,13 @@ class Block():
     def __init__(self):
         self.width = 32
         self.addr_width = 12
-        self.base_addr = 0
+        self.base_addr =  OrderedDict()
```

Comment:
Does this have to be an ordered dict?

Reply:
do you suggest to use regular dict? I thought OrderedDict is preferred.